# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/manager/ssl/components.conf
# Commit: 2b647eb982bc
# Full Hash: 2b647eb982bc0068d39fe58c6f59bebe55909343
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2023-06-07 21:43:58
# Regressor Bug: 1834862
# File Overlap Count: 1
# Description:
#   Bug 1834862 - implement nsICryptoHash in rust r=jschanck,supply-chain-reviewers
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D179727
# ==============================================================================

diff -r 7c11166969ff -r 2b647eb982bc security/manager/ssl/components.conf
--- a/security/manager/ssl/components.conf	Wed Jun 07 15:45:14 2023 +0000
+++ b/security/manager/ssl/components.conf	Wed Jun 07 15:56:37 2023 +0000
@@ -69,8 +69,8 @@
     {
         'cid': '{36a1d3b3-d886-4317-96ff-87b0005cfef7}',
         'contract_ids': ['@mozilla.org/security/hash;1'],
-        'type': 'nsCryptoHash',
-        'legacy_constructor': 'mozilla::psm::NSSConstructor<nsCryptoHash>',
+        'headers': ['/security/manager/ssl/crypto_hash/crypto_hash.h'],
+        'legacy_constructor': 'crypto_hash_constructor',
     },
     {
         'cid': '{45a5fe2f-c350-4b86-962d-02d5aaaa955a}',